core/state_processor.go

```diff
@@ -293,11 +293,11 @@ func (p *StateProcessor) Process(block *types.WorkObject, batch ethdb.Batch) (ty
 	time3 := common.PrettyDuration(time.Since(start))
 
 	// Iterate over and process the individual transactions.
-	etxRLimit := len(parent.Transactions()) / params.ETXRegionMaxFraction
+	etxRLimit := (len(parent.Transactions()) * 21000) / params.ETXRegionMaxFraction
```

Comment:
Can you use params.ETXGas instead of hardcoding

Reply:
Done